### DISCUSSION TITLE
请问 图片与表格的中文标点符号居中怎么解决

### DISCUSSION THREAD

**Discussion by Qi-Yangsjtu** [sjtug/SJTUThesis#769] [提问]
![latex](https://user-images.githubusercontent.com/53613122/154970335-a67b2749-8889-4785-aa06-6f51b0796469.png)

**Comment by skyzh** [comment]
你使用的楷体（AR-PL-UKai，台湾文鼎）不符合大陆的书写习惯，可以换一套字体。

参考 https://github.com/sjtug/SJTUThesis/discussions/617

**Reply by Qi-Yangsjtu** [reply]
问题解决了 非常感谢！
